sniffles/rulereader.py

```diff
@@ -1263,6 +1271,9 @@ def parseRuleFile(self, filename=None):
 
                 if 'typets' in ts.attrib:
                     typeRuleTS = ts.attrib['typets']
+                    if typeRuleTS == "BackgroundTraffic":
+                        self.background_percent = ts.attrib['percentage']
+                        continue
```

Comment:
I don't know why we need to continue.  Can't we set various values for a bg-traffic stream, too?  Like ip src and dst, and so on.  Is it possible to make a Petabi rule, also be a bg-traffic rule?